wrk_minidump.lua

```diff
@@ -0,0 +1,29 @@
+-- This is a `wrk` script that posts a multipart minidump request to symbolicator.
```

Comment:
could we not have this in the root of the project somehow?  maybe a "scripts" or "contrib" directory?